packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -300,6 +300,11 @@ export class CdpAutomation {
           return this.getCookie(data)
         })
 
+      case 'add:cookies':
+        setCookie = data.map((cookie) => normalizeSetCookieProps(cookie))
```

Comment:
can we add the `Protocol.Network.SetCookieRequest[]` type to `setCookie`?

Reply:
Yep. Addressed in [`db8684a` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/db8684ab6c3f49fda2961344dcb23972701465b9)